src/Microsoft.AspNetCore.Mvc.Core/ApplicationModels/ParameterTransformerConvention.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Mvc.ApplicationModels;
+using Microsoft.AspNetCore.Routing;
+
+namespace Microsoft.AspNetCore.Mvc.Core.ApplicationModels
+{
+    public sealed class RouteTokenTransformerConvention : IApplicationModelConvention
```

Comment:
This doesn't need to come with MVC. Can be something that people who want to use a transformer could include with there app

Reply:
I think this is pretty nice.